cachito/web/api_v1.py

```diff
@@ -196,5 +200,10 @@ def patch_request(request_id):
                 request.dependencies.append(dep_obj)
 
     db.session.commit()
+    if current_user.is_authenticated:
+        flask.current_app.logger.info(
+            'The user %s patched request %d', current_user.username, request.id)
```

Comment:
What's `current_user.username` for anonymous users?

Reply:
It raises an `AttributeError`.

Reply:
Bummer... Ideally, this would be set to something reasonable such as `None`. I don't necessarily like the conditional logging. It's easy to forget to do it in all places. But I don't see a way around it.